CMakeLists.txt

```diff
@@ -180,7 +180,6 @@ if (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} GREATER 3.5) # Need 3.6 or abo
       ",-google-runtime-int"
       ",-google-runtime-references"
       ",-misc-misplaced-widening-cast"
-      ",-misc-unused-parameters"
```

Comment:
Why remove the check?

Reply:
All checks are enabled, but the listed ones, so I simply re-enable `misc-unused-parameters` 👍

Reply:
OK for you @rfjakob ? Thx 👍